index.js

```diff
@@ -1,29 +1,7 @@
-var path = require("path");
-var fs = require("fs");
-var url = require("url");
-
-// We want to load Readability and JSDOMParser, which aren't set up as commonjs libraries,
-// and so we need to do some hocus-pocus with 'vm' to import them on a separate scope
-// (identical) scope context.
-var vm = require("vm");
-var readabilityPath = path.join(__dirname, "Readability.js");
-var jsdomPath = path.join(__dirname, "JSDOMParser.js");
-
-
-var scopeContext = {};
-// We generally expect dump() and console.{whatever} to work, so make these available
-// in the scope we're using:
-scopeContext.dump = console.log;
-scopeContext.console = console;
-scopeContext.URL = url.URL;
```

Comment:
The [URL class](https://nodejs.org/api/url.html#url_class_url) has been added to Node 10.

Reply:
Yes, the required node version should be reflected in `package.json`, if we decide that Node 10 is a good minimum version. Looking at [the releases](https://github.com/nodejs/release#release-schedule), it seems aligned with their support policy. 

Alternatively, something like:

```js
if (typeof URL === 'undefined' && typeof require === 'function') {
  URL = require('url').URL;
}
```

would allow us to use the legacy URL parsing functionality in earlier Node.js versions.

Reply:
Making the requirement 10.0 seems fine to me. Thanks!